### PR TITLE
CY-1902 Update the db_nodes table when adding or removing a node

### PR DESCRIPTION
**Pull request by inbala**

### CODE REVIEW

File: cfy_manager/components/postgresql_server/postgresql_server.py [hunk 1]
```diff
@@ -1038,7 +1038,43 @@ def _restart_manager_db_dependent_services(self):
         common.sudo(['systemctl', 'restart', 'cloudify-amqp-postgres'])
         common.sudo(['systemctl', 'restart', 'cloudify-restservice'])
 
-    def add_cluster_node(self, address):
+    def _node_is_in_db(self, node_id):
```
Comment (tyacbovi) [inline cfy_manager/components/postgresql_server/postgresql_server.py]: I would here just return the num with the same node_id, and move the logic of ==1 to the add command

File: cfy_manager/main.py [hunk 2]
```diff
@@ -329,12 +335,16 @@ def db_node_list(**kwargs):
                      default=False)
 @argh.decorators.arg('-a', '--address', help=DB_NODE_ADDRESS_HELP_MSG,
                      required=True)
+@argh.decorators.arg('-i', '--node-id', help=DB_NODE_ID_HELP_MSG,
+                     required=True)
+@argh.decorators.arg('-n', '--hostname', help=DB_HOSTNAME_HELP_MSG)
```
Comment (tyacbovi) [inline cfy_manager/main.py]: =>-h (-H)

File: cfy_manager/components/postgresql_server/postgresql_server.py (hunk 1 above)
Comment (geokala) [reply cfy_manager/components/postgresql_server/postgresql_server.py]: That would only make sense if this function weren't called '_node_is_in_db'. Is there a benefit to splitting this logic?

File: cfy_manager/main.py [hunk 3]
```diff
@@ -343,12 +353,14 @@ def db_node_add(**kwargs):
                      default=False)
 @argh.decorators.arg('-a', '--address', help=DB_NODE_ADDRESS_HELP_MSG,
                      required=True)
+@argh.decorators.arg('-i', '--node-id', help=DB_NODE_ID_HELP_MSG,
```
Comment (geokala) [inline cfy_manager/main.py]: Isn't this only required if running the command on a manager?

File: cfy_manager/main.py (hunk 2 above)
Comment (inbala) [reply cfy_manager/main.py]: I think -h is taken for help

File: cfy_manager/main.py (hunk 3 above)
Comment (inbala) [reply cfy_manager/main.py]: Yes.
So it runs also on the db nodes?

File: cfy_manager/main.py (hunk 2 above)
Comment (geokala) [reply cfy_manager/main.py]: It is.

File: cfy_manager/main.py (hunk 3 above)
Comment (geokala) [reply cfy_manager/main.py]: The remove is, yes.

**Pull request merged**